src/templates/language_packs.html

```diff
@@ -0,0 +1,46 @@
+{% from "_macros/market_button.html" import market_button %}
+{% include '_macros/more_button.html' %}
+
+<section class="main">
```

Comment:
Cool. I tried to re-use `app_list.html`, but it did a few things I didn't like: cast to model cache (unneeded here, I don't even have a detail page), include app filters, full app tile.